style.css

```diff
@@ -1,15 +1,343 @@
+
+.under-head-meal-section {
+  text-align: center;
+  font-size: 0.9em;
+}
+
+.about-you {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 4em;
+}
+.vege-card {
+  background-color: #ffffff;
+  width: 27%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.rotateimg180 {
+  transform: rotate(90deg);
+}
+.title-about-you {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  text-align: center;
+  margin: 2em 0.5em;
+}
+.vege-text {
+  font-size: 0.75em;
+  margin-top: 1em;
+}
+
+.meals {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 6em;
+}
+
+.refugees {
+  background-color: #ffffff;
+  width: 27%;
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: left;
+  margin-top: 3em;
+}
+
+.refugees-title {
+  color: #0a0a0a;
+  font-size: 2.2em;
+  text-align: left;
+  font-weight: bold;
+  line-height: 1em;
+  margin-left: 0.15em;
+}
+
+.text-refugees {
+  font-size: 0.75em;
+  margin: 1.5em 0.5em 4em 0.5em;
+}
+
+.meal-image {
+  color: #0a0a0a;
+  justify-content: center;
+  align-items: left;
+  margin: 0em 0.5em;
+}
+
+.button-contact {
+  margin: 1em 0;
+  border: 1px solid #e31c3a;
+  border-radius: 4px;
+  background-color: #e31c3a;
+  padding: 0.8em;
+  color: #ffffff;
+  outline: none;
+  cursor: pointer;
+  margin-left: 0.26em;
+  text-transform: uppercase;
   width: 60%;
+}
+
+.meal-block {
   background-color: #ffffff;
-  height: 100vh;
+  width: 27%;
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  align-items: left;
+}
+
+.price-with-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  align-items: left;
+  margin: 1.5em 0.5em 0.5em 3.5em;
+}
+.price {
+  display: flex;
+  flex-direction: row;
+  font-size: 2em;
+  font-weight: bold;
+}
+
+.dolar {
+  font-size: 0.7em;
+}
+
+.recipe-title {
+  font-size: 1.5em;
+  text-transform: uppercase;
+  margin-bottom: 0.5em;
+}
+.recipe-text {
+  font-size: 0.75em;
+}
+
+.third-section {
+  display: flex;
+  flex-direction: row;
+  margin-top: 5em;
+}
+
+.meal-photo {
+  display: flex;
+  width: 53%;
+  justify-content: center;
+  margin-left: -1em;
+}
+
+.city {
+  display: flex;
+  flex-direction: column;
+  width: 47%;
+  justify-content: center;
+}
+
+.city-title {
+  color: #0a0a0a;
+  font-size: 2.2em;
+  text-align: left;
+  font-weight: bold;
+  line-height: 1em;
+  padding-left: 25%;
+}
+
+.text-city {
+  font-size: 0.75em;
+  padding-left: 25%;
+  padding-top: 5%;
+
+  margin-right: 6em;
+}
+
+.button-request {
+  margin-left: 25%;
+  margin-top: 2em;
+  border: 1px solid #e31c3a;
+  border-radius: 4px;
+  background-color: #e31c3a;
+  color: #ffffff;
+  outline: none;
+  cursor: pointer;
+  padding: 0.8em;
+  width: 45%;
+  text-transform: uppercase;
+}
+
+footer {
+  margin: 5em 0em;
+  display: flex;
+  flex-direction: column;
+  padding: 0em 28%;
```

Comment:
Avoid using percentages for padding and margin if you may. We will talk about this subject in the meeting to discuss it further